File: python/14a_41A_Translation.py
def main():
    s = input()
    t = input()
    print(func(s, t))

def func(s,t):
    temp = ""
    for k in range(len(s)-1, -1, -1):
        temp += s[k] 
    return "YES" if temp == t else "NO"

if __name__ == "__main__":
    main()

File: cpp/14a_41A_Translation.cpp
#include<bits/stdc++.h>

int main(){
    std::string s, t, temp;
    std::cin>>s>>t;
    temp = s;
    reverse(s.begin(), s.end());
    if (s.compare(t)){
        std::cout<<"NO";
    }
    else{
        std::cout<<"YES";
    }
    return 0;
}